src/commands/registries/azure/DockerWebhookCreateStep.ts

```diff
@@ -0,0 +1,83 @@
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import ContainerRegistryManagementClient from 'azure-arm-containerregistry';
+import { Webhook, WebhookCreateParameters } from 'azure-arm-containerregistry/lib/models';
+import { Site } from 'azure-arm-website/lib/models';
+import { Progress } from "vscode";
+import * as vscode from "vscode";
+import { IAppServiceWizardContext, SiteClient } from "vscode-azureappservice";
+import { AzureWizardExecuteStep, createAzureClient, IActionContext } from "vscode-azureextensionui";
+import { ext } from "../../../extensionVariables";
+import { AzureRegistryTreeItem } from '../../../tree/registries/azure/AzureRegistryTreeItem';
+import { AzureRepositoryTreeItem } from '../../../tree/registries/azure/AzureRepositoryTreeItem';
+import { DockerHubRepositoryTreeItem } from '../../../tree/registries/dockerHub/DockerHubRepositoryTreeItem';
+import { RemoteTagTreeItem } from '../../../tree/registries/RemoteTagTreeItem';
+import { nonNullProp } from "../../../utils/nonNull";
+import { openExternal } from '../../../utils/openExternal';
+
+export class DockerWebhookCreateStep extends AzureWizardExecuteStep<IAppServiceWizardContext> {
+    public priority: number = 141; // execute after DockerSiteCreate
+    private _treeItem: RemoteTagTreeItem;
+    public constructor(treeItem: RemoteTagTreeItem) {
+        super();
+        this._treeItem = treeItem;
+    }
+
+    public async execute(context: IAppServiceWizardContext, progress: Progress<{
+        message?: string;
+        increment?: number;
+    }>): Promise<void> {
+        const site: Site = nonNullProp(context, 'site');
+        let siteClient = new SiteClient(site, context);
+        let appUri: string = (await siteClient.getWebAppPublishCredential()).scmUri;
+        if (this._treeItem.parent instanceof AzureRepositoryTreeItem) {
+            const creatingNewWebhook: string = `Creating webhook for web app "${context.newSiteName}"...`;
+            ext.outputChannel.appendLine(creatingNewWebhook);
+            progress.report({ message: creatingNewWebhook });
+            const webhook = await this.createWebhookForApp(this._treeItem, context, appUri);
+            ext.outputChannel.appendLine(`Created webhook '${webhook.name}' with scope '${webhook.scope}', id: '${webhook.id}' and location: '${webhook.location}'`);
+        } else if (this._treeItem.parent instanceof DockerHubRepositoryTreeItem) {
+            // point to dockerhub to create a webhook
+            // http://cloud.docker.com/repository/docker/<registryName>/<repoName>/webHooks
+            const dockerhubPrompt: string = "Copy & open";
+            const dockerhubUri: string = `https://cloud.docker.com/repository/docker/${this._treeItem.parent.parent.parent.username}/${this._treeItem.parent.repoName}/webHooks`;
```

Comment:
Could you update the screenshot with the new popup?

Reply:
![image](https://user-images.githubusercontent.com/30420973/61986427-ec552f80-afc3-11e9-87ac-264008491edf.png)